cli/command/cli.go

```diff
@@ -135,9 +136,12 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 	if err != nil {
 		return errors.Wrap(err, "Experimental field")
 	}
+	orchestrator := GetOrchestrator(hasExperimental, opts.Common.Orchestrator, cli.configFile.Orchestrator)
 	cli.clientInfo = ClientInfo{
 		DefaultVersion:  cli.client.ClientVersion(),
 		HasExperimental: hasExperimental,
+		HasKubernetes:   hasExperimental && orchestrator == OrchestratorKubernetes,
```

Comment:
Since this is a derived value I think it would be better to expose it as a method `HasKubernetes()` instead of storing the derived value.

Reply:
Fixed